# Generated from /home/pmukherj/outdoor_test5.bag on 2013-09-09.
mag_zero_x: 0.000000
mag_zero_y: -0.000000
mag_zero_z: 0.000000
mag_zero_radius: 0.000000
